<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelListWidgetUI</class>
 <widget class="QWidget" name="ChannelListWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel List</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="13" column="0" colspan="5">
     <widget class="QLabel" name="m_statsLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="m_minUser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="whatsThis">
       <string>You can limit the channel list to those channels with a minimum number of users here. Choosing 0 disables this criterion.</string>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="m_maxUser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="whatsThis">
       <string>You can limit the channel list to those channels with a maximum number of users here. Choosing 0 disables this criterion.</string>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="m_lblMaxUser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Ma&amp;ximum users:</string>
      </property>
      <property name="buddy">
       <cstring>m_maxUser</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="m_lblMinuser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Minimum users:</string>
      </property>
      <property name="buddy">
       <cstring>m_minUser</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="m_lblFilter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Filter pattern:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="5">
     <widget class="KLineEdit" name="m_filterLine">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Enter a filter string here.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="m_channelBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Channel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QCheckBox" name="m_topicBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Topic</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="8">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="m_refreshListBtn">
        <property name="text">
         <string>Refresh List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_saveListBtn">
        <property name="text">
         <string>Save &amp;List...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_joinChanBtn">
        <property name="whatsThis">
         <string>Click here to join the channel. A new tab is created for the channel.</string>
        </property>
        <property name="text">
         <string>&amp;Join channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="8">
     <widget class="QTreeWidget" name="m_channelListView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>The filtered list of channels is displayed here. Notice that if you do not use regular expressions, Konversation lists any channel whose name contains the filter string you entered. The channel name does not have to start with the string you entered.\n\nSelect a channel you want to join by clicking on it. Right click on the channel to get a list of all web addresses mentioned in the channel's topic.</string>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Channel Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Users</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Channel Topic</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="13" column="5">
     <widget class="QLabel" name="m_bufferLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Channel List Buffer:</string>
      </property>
      <property name="buddy">
       <cstring>m_buffer</cstring>
      </property>
     </widget>
    </item>
    <item row="13" column="6" colspan="2">
     <widget class="QProgressBar" name="m_buffer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%v/%m</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="m_regexBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Regular expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_applyBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Click here to retrieve the list of channels from the server and apply the filter.</string>
        </property>
        <property name="text">
         <string>&amp;Apply Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
